app/javascript/components/miq-toolbar.jsx

```diff
@@ -45,6 +45,40 @@ const getParams = (urlParms, sendChecked) => {
   return _.filter(params).join('&') || undefined;
 };
 
+const anotherArrTest = (fullParamString) => {
+  // Turn full string into an array holding each key-value pair
+  // selectedUser=all&approvalStateCheckboxes=approved&approvalStateCheckboxes=denied&types=all&selectedPeriod=7&reasonText=
```

Comment:
please remove extra comments.  You can write couple to lines to explain the function.